src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/** A chatbot that handles todo-list operations. */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * A constructor for the class Duke.
+     *
+     * @param filePath The relative path of the file that stores the user's task list.
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Execute the chatbot functions based on user's command.
```

Comment:
I believe Javadoc headers should start with words in the form `Returns ...`, `Sends ...`, `Adds ...` (not `Return` or `Returning` etc.)
This applies to your other javadocs headers  too :)